### PR TITLE
Adds ores

### PR DESCRIPTION
**Pull request by utahclock**
Adds Ores

### CODE REVIEW

**Comment by oisin100** [comment]
Woops!

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -176,8 +176,6 @@
 		src.loc.visible_message("The [src] begins to shake. Stand back!")
 		used = TRUE
 		sleep(50)
-		message_admins("A survival pod has been activated at [loc.x],[loc.y],[loc.z] (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[loc.x];Y=[loc.y];Z=[loc.z]'>JMP</A>) last touched by [fingerprintslast]")
```
Comment (oisin100) [inline code/modules/mining/mine_items.dm:179]: Why are you removing this?

File: code/modules/mining/equipment_locker.dm [hunk 2]
```diff
@@ -19,7 +19,8 @@
 	var/ore_pickup_rate = 15
 	var/sheet_per_ore = 1
 	var/point_upgrade = 1
-	var/list/ore_values = list(("sand" = 1), ("iron" = 1), ("gold" = 20), ("silver" = 20), ("uranium" = 20), ("bananium" = 30), ("diamond" = 40), ("plasma" = 40))
+	var/list/ore_values = list(("sand" = 1), ("iron" = 1), ("gold" = 20), ("silver" = 20), ("uranium" = 20), ("bananium" = 30), ("diamond" = 40), ("plasma" = 40), ("coal" = 40), ("copper" = 40),
+	("clarenkite" = 40), ("electrite" = 40), ("platnium" = 40))
```
Comment (oisin100) [inline code/modules/mining/equipment_locker.dm]: Bit strange moving it on to its own line... Not really important though.

File: code/modules/mining/mine_items.dm (hunk 1 above)
Comment (utahclock) [inline code/modules/mining/mine_items.dm:179]: i didnt remove it i think my push revereted it ... as it seems to have pushed 75 something files when it should have only done like 8 files

**Pull request closed without merging**